.changeset/dry-cherries-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-shell': minor
+---
+
+Change the DOM structure for portal container
```

Comment:
Note that the information we write here should ideally help users that read the changelog to understand what changed.

In this case, this sentence essentially tells what technically changed but without any further context it's hard to make a sense of it.

Instead, we can mention what kind of issue we fixed. For example:

```suggestion
Page notifications (like errors) now correctly push the page content below them, even for modal pages.
```

Reply:
Thanks for the explanation😊.